v0.7/speech_recognition/rnnt/run.sh

```diff
@@ -38,29 +47,40 @@ if [[ $stage -le 0 ]]; then
   wget https://zenodo.org/record/3662521/files/DistributedDataParallel_1576581068.9962234-epoch-100.pt?download=1 -O $work_dir/rnnt.pt
 fi
 
-# stage 1: download data. Check checksum to skip?
+# stage 1: download data. This will hae a non-zero exit code if the
+# checksum is incorrect.
 if [[ $stage -le 1 ]]; then
-  mkdir -p $local_data_dir
-  mkdir -p $librispeech_download_dir
-  python utils/download_librispeech.py \
-         utils/librispeech-inference.csv \
+  python pytorch/utils/download_librispeech.py \
+         pytorch/utils/librispeech-inference.csv \
          $librispeech_download_dir \
          -e $local_data_dir
 fi
 
 if [[ $stage -le 2 ]]; then
-  python utils/convert_librispeech.py \
+  python pytorch/utils/convert_librispeech.py \
       --input_dir $librispeech_download_dir/dev-clean \
       --dest_dir $local_data_dir/dev-clean-wav \
       --output_json $local_data_dir/dev-clean-wav.json
 fi
 
 if [[ $stage -le 3 ]]; then
-   python inference.py \
-      --model_toml configs/rnnt.toml \
+    
+    # python run.py --backend pytorch \
+    #        --dataset_dir $local_data_dir \
+    #        --manifest $local_data_dir/dev-clean-wav.json \
+    #        --pytorch_config_toml pytorch/configs/rnnt.toml \
+    #        --pytorch_checkpoint pytorch/work_dir/rnnt.pt \
+    #        --scenario SingleStream \
+    #        --accuracy
+
+    # ipython --pdb -c "%run \
+    # python \
+    gdb --ex run --args python \
+    pytorch/inference.py \
```

Comment:
Eventually this will be deleted and the above "python run.py" (which uses loadgen) line will be uncommented.